core/src/main/java/com/megaease/easeagent/core/Bootstrap.java

```diff
@@ -93,9 +93,10 @@ public static void start(String args, Instrumentation inst, Iterable<Class<?>> p
         port = Integer.parseInt(portStr);
         AgentHttpServer agentHttpServer = new AgentHttpServer(port);
         agentHttpServer.addHttpRoutes(AGENT_HTTP_HANDLER_LIST_ON_INIT);
-        agentHttpServer.startServer();
-        LOGGER.info("start agent http server on port:{}", port);
-
+        if (port > 0) {
```

Comment:
I don't think it's a good idea, disabling HTTP server deserves a dedicated configuration item,such as:
```
easeagent.server.enabled = false (default is True)
```

HTTP server should be enabled by default, if a user forgets to config a port or config an invalid port, we should use the default port to boot the HTTP server.

Reply:
I fixed it. Add configuration `easeagent.server.enabled = true`